blog/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+from blog.models import Commentary
+
+
+class CommentaryForm(forms.ModelForm):
+    class Meta:
+        model = Commentary
+        fields = ["content"]
```

Comment:
Consider using crispy forms to enhance the appearance of your form. You can do this by adding `from crispy_forms.helper import FormHelper` and `from crispy_forms.layout import Submit` at the top, and then initializing a `FormHelper` instance in the form's `__init__` method. This aligns with the task requirement to use crispy forms for better aesthetics.